src/Git/Commit.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace F3\Changelog\Git;
+
+class Commit
+{
+    /**
+     * @var string
+     */
+    private $hash;
+
+    /**
+     * @var \DateTIme
+     */
+    private $date;
+
+    /**
+     * @var string
+     */
+    private $subject;
+
+    public static function fromLogEntry(array $logEntry): self
```

Comment:
Actually hydrator or, in case of example above, static factory method should validate input data. Or we could just add private constructor which will handle this checks for us.

> Is that on purpose?

This is just a demonstration of idea. So the reason for this is just that I'm to lazy